File: get_request.dart
// Random Dart function generated on 2025-06-17

class GetData {
  final int id;
  final String name;
  final double value;

  const GetData({
    required this.id,
    required this.name,
    required this.value,
  });
}

List<GetData> get_data(List<GetData> items) {
  const double multiplier = 3.0;
  
  return items
      .map((item) => GetData(
            id: item.id,
            name: '${item.name}_processed',
            value: item.value * multiplier,
          ))
      .toList();
}

void main() {
  final testData = [
    GetData(id: 1, name: 'Item1', value: 54.0),
  ];

  final result = get_data(testData);
  print('Processed ${result.length} items');
}

File: get_request.go
// Random Go function generated on 2025-08-29
package main

import (
    "fmt"
    "time"
)

type Item struct {
    ID        int       `json:"id"`
    Name      string    `json:"name"`
    Value     int       `json:"value"`
    CreatedAt time.Time `json:"created_at"`
}

func get_item(items []Item) []Item {
    multiplier := 9
    result := make([]Item, len(items))
    
    for i, item := range items {
        result[i] = Item{
            ID:        item.ID,
            Name:      fmt.Sprintf("%s_processed", item.Name),
            Value:     item.Value * multiplier,
            CreatedAt: time.Now(),
        }
    }
    
    return result
}

func main() {
    testData := []Item{
        {ID: 1, Name: "item1", Value: 78, CreatedAt: time.Now()},
        {ID: 2, Name: "item2", Value: 85, CreatedAt: time.Now()},
    }
    
    result := get_item(testData)
    fmt.Printf("Processed %d items\n", len(result))
}
